Cargo.toml

```diff
@@ -35,4 +35,5 @@ priority = "optional"
 assets = [
     ["target/release/git-interactive-rebase-tool", "usr/bin/interactive-rebase-tool", "755"],
     ["README.md", "usr/share/doc/cargo-deb/README", "644"],
+    ["git-interactive-rebase-tool.1", "usr/share/man/man1/git-iteractive-rebase-tool.1", "644"]
```

Comment:
`usr/share/man/man1/git-iteractive-rebase-tool.1` to `usr/share/man/man1/iteractive-rebase-tool.1` please